### DISCUSSION TITLE
Add clear call to action to the "anti ctb" popup

### DISCUSSION THREAD

**Discussion by QbieShay** [sn4k3/UVtools#962] [Ideas]
Hello! New to the 3d printing community, but not to open source.

I entirely agree with the principle of moving away from closed formats, and i really appreciate that uvtools tries to make users aware of it.

I'm finding it a bit difficult, from a user standpoint, to find the proper way to do so. I was given a 3d printer from a friend, and he setup lychee slicer for me. I tried to learn prusa but the auto support didn't seem as good as lychee, and in general i had trouble setting everything up.

I find that the popup is missing a concrete call to action for people like me, that aren't really familiar with the 3d printing world. How would i support open formats? How would i participate in pressuring 3d printer manufacturers and slicer developers to move away from their proprietary formats?

I am hoping to get some advice for me personally, but also to find ways to improve the popup to increase the chance to mobilitate people. 

Cheers!

**Comment by sn4k3** [comment]
Unfortonally market is still at proprietary and closed formats.
I tried to push mSLA to klipper: https://github.com/Klipper3d/klipper/pull/6575 but the lack of testers and adoption make it hard to happen
